docs/wiki/feature/medical-ai.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: wiki
+title: Medical AI
+description: Makes AI heal themselves and each other with the ACE Medical system.
+group: feature
+category: realism
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 7
+  patch: 0
+---
+
```

Comment:
Ugh, I wrote that before I fully read it and forgot to remove it. Note stays, should add it at the top for quick info look.